src/message/socket-wrapper.js

```diff
@@ -40,6 +42,53 @@ var SocketWrapper = function( socket, options ) {
 };
 
 utils.inherits( SocketWrapper, EventEmitter );
+SocketWrapper.lastPreparedMessage = null;
+
+/**
+ * Updates lastPreparedMessage and returns the [uws] prepared message.
+ *
+ * @public
+ * @returns {External} prepared message
+ */
+SocketWrapper.prepareMessage = function( message ) {
+	SocketWrapper.lastPreparedMessage = message;
+	return uws.native.server.prepareMessage( message, uws.OPCODE_TEXT );
+}
+
+/**
+ * Sends the [uws] prepared message, or in case of testing sends the
+ * last prepared message.
+ *
+ * @public
+ * @returns {void}
+ */
+SocketWrapper.prototype.sendPrepared = function(preparedMessage) {
+	if ( this.socket.external ) {
```

Comment:
First check is if this socket is valid aka not disconnected. Then the next is a check if external is not null (which means it is undefined instead of null which means it is not an uws socket)